lib/linguist/heuristics.yml

```diff
@@ -329,6 +329,8 @@ disambiguations:
     pattern: '^\s+\w+\s+=>\s'
 - extensions: ['.pro']
   rules:
+  - language: Proguard
+    pattern: '^-(include\b.*\.pro|keep\b|keepclassmembers\b|keepattributes\b)'
```

Comment:
Maybe we could anchor `\.pro` to the end of line? It might reduce backtracking.

Reply:
Done :+1: